pkcs11-env.h

```diff
@@ -29,6 +29,24 @@
 #define NULL_PTR 0
 #endif
 
+#ifdef PKCS11_LONGTYPE
+ #include <stdint.h>
+ #if PKCS11_LONGTYPE==32
```

Comment:
nit: calling this ".._LONGTYPE" feels slightly confusing &ndash; based on the name I'd guess that `-DPKCS11_LONGTYPE=int32_t` would be how to set it.  Maybe rename to `PKCS11_LONG_SIZE` ?

That would also make the difference between `PKCS11_LONGTYPE` and `PKCS11_LONG_TYPE` clearer :-)

Reply:
I like PKCS11_LONG_SIZE, excellent suggestion.